internal/gengapic/doc_file.go

```diff
@@ -111,6 +111,7 @@ func (g *generator) genDocFile(year int, scopes []string, serv *descriptor.Servi
 	p("import (")
 	p("%s%q", "\t", "context")
 	if hasREST {
+		p("%s%q", "\t", "fmt")
```

Comment:
I think your rebase got kind of messed up because this was already commited and shouldn't show up in the diff

Reply:
holy cow I'm sorry my git got totally messed up. I think it's fixed now, and only files that should be getting changed are actually being changed in this PR. I'm sorry >.<